src/index.js

```diff
@@ -11,12 +11,12 @@ import PulseEngine from './PulseEngine';
 const port = +process.env.PORT || 3050;
 
 process.on('uncaughtException', err => {
-  console.error(err); // eslint-disable-line no-console
+  console.error(err.toString()); // eslint-disable-line no-console
```

Comment:
Wouldn't `console.error` automatically trigger the `toString` method if it is an object?